Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -61,8 +61,8 @@ Write-Host "packageFile= $DacpacFilePath"
 #Ensure that a single package (.dacpac) file is found
 ThrowIfMultipleFilesOrNoFilePresent -files $DacpacFilePath -pattern $DacpacFile
 
-$PublishProfilePath=""
-if( !([string]::IsNullOrWhitespace($PublishProfile)) )
+$PublishProfilePath = ""
+if( [string]::IsNullOrWhitespace($PublishProfile) -eq $false -and $PublishProfile -ne $env:SYSTEM_DEFAULTWORKINGDIRECTORY )
```

Comment:
I remember DeployRG doing one more check comparing against $env:SYSTEM_DEFAULTWORKINGDIRECTORY/ with a / in end.